### DISCUSSION TITLE
remove the green markers from maps (if it possible)

### DISCUSSION THREAD

**Discussion by adalmo** [PaperMC/Paper#10554] [Ideas]
### Is your feature request related to a problem?

it killing fps / server perfomances & get player higher pings when someone tried to place item frames with normal maps as many as & hold the map & also getting some glitches <how is it works> -> https://www.youtube.com/watch?v=W12zKDvHsQI&ab_channel=FitMC (i can't post my own video cuz of due limits of file size)

### Describe the solution you'd like.

i hope this one should be fix able or i need to find other way to prevent em (but seems like it doesn't has some kind of plugins to prevent it)

### Describe alternatives you've considered.

remove green markers from map

**Comment by electronicboy** [comment]
Mojang has already added a limit to how many decorations a map can have, I don't think that we'll explicitly add a toggle for this, but, maybe a config to change the limit?

**Comment by adalmo** [comment]
hmm but somehow i tried to keep place it and it drops fps like 17 while without em was 300+ (when i look behind fps goes like normal but when i turn around to maps it drops to 16) 
so ig i can try to limit the item frames per 1 chunk but not enough to stop it.. if there is a config to limit it idk which one it does (also what it cause the fps lags from thoes maps not the item frames)
![2022-10-13_08 16 01](https://user-images.githubusercontent.com/88581750/195483642-8320acdb-6b37-4c75-9076-8a63b1f9e064.png)

**Comment by adalmo** [comment]
seems like thoes green markers are glitching
https://user-images.githubusercontent.com/88581750/195484868-28485c92-fe94-4708-bf3f-d55efcc402dc.mp4

**Comment by Doc94** [comment]
This merged PR already handle this?
https://github.com/PaperMC/Paper/pull/8545

**Comment by Owen1212055** [comment]
In general I think that this is better for some kind of config option, where basically this functionality can be disabled entirely.

**Comment by Warriorrrr** [comment]
Isn't this possible by setting the map cursor limit option to 0?
